MergerCli/SourceParser.cs

```diff
@@ -91,7 +91,8 @@ private IData ParseFileSource(string[] args, ref int idx, int batchSize, bool is
 
             }
             idx += paramCount;
-            return this._dataFactory.CreateDataSource(sourceType, sourcePath, batchSize, isOneXOne, origin, null, isBase);
+            Grid grid = isOneXOne ? Grid.OneXOne : Grid.TwoXOne;
```

Comment:
this bypass the default in the data source

Reply:
Opened another issue to fix this.